README.rst

```diff
@@ -20,62 +20,100 @@ Overview
+
+  ::
+
     pip3 install -e ".[dev]"
     pip3 install --editable .
-    # start etcd in container
-    docker-compose up -d # start
+
+- Start docker container with etcd storage
+
+  ::
+
+    docker-compose up -d
+
+- You can start KQueen API service directly
+
+  ::
+
     kqueen
 
--  Clean etcd storage and prepare examples
+- Prepare kubernetes config file
 
-`devenv.py` will create few objects to provides basic developer environment. It will also try to download `kubeconfig` file for real cluster but it requires access to Mirantis VPN. However, it can be workarounded by creating file `kubeconfig_remote` in repository root and this file will be used instead of downloading it.
+ Kubernetis configuration file that describes existing multiple clusters can be used to Kqueen.
+ Rename it with *kubernetis_remote* and placed to the root of the project.
+ For test purposes this file can be empty, but should be added manually.
 
-::
 
-    # exec in kqueen-api container
+How-to's
+^^^^^^^^
+
+
+- Clean etcd storage after previous runs
+
+  ::
+
     etcdctl rm --recursive /kqueen
+
+- Add admin user, organization, mock clusters and provisioners to etcd storage at once, execute the following
+
+  ::
+
     ./devenv.py
 
-- Run flask shell
+- To add a single *admin* user with *default* password within associated *DemoOrg* organization in provided *demoorg* namespace, execute the following
 
-::
+  ::
 
-    # exec in kqueen-api container
-    export FLASK_APP=kqueen.server
-    export prometheus_multiproc_dir=$(mktemp -d)
-    flask shell
+    ./bootstrap_admin.py DemoOrg demoorg admin default
 
-- Test access token with `curl`
+- Test access token. *curl*,  *jq* should be installed in your system
 
-::
+  ::
 
-    TOKEN=$(curl -s -H "Content-Type: application/json" --data '{"username":"admin","password":"default"}' -X POST localhost:5000/api/v1/auth | jq -r '.access_token'); echo $TOKEN; curl -H "Authorization: Bearer $TOKEN" localhost:5000/api/v1/clusters
+    TOKEN=$(curl -s -H "Content-Type: application/json" --data '{"username":"admin","password":"default"}' -X POST localhost:5000/api/v1/auth | jq -r '.access_token')
+    echo $TOKEN
+    curl -H "Authorization: Bearer $TOKEN" localhost:5000/api/v1/clusters
 
-- Image updating
+- Set up flask shell for manual testing and debugging
 
-There are two ways to test development changes. Its possible to create a separate branch and push PR, then TravisCI build image and push it on Docker Hub automatically. Or just rebuild kqueen api-image locally:
+  ::
 
-::
+    export FLASK_APP=kqueen.server
+    export prometheus_multiproc_dir=$(mktemp -d)
+    flask shell
+
+- Update Doker image with code changes
+
+There are two ways to test development changes. First is automatic: create a separate branch and push PR, then TravisCI
+build image and push it on Docker Hub automatically. Second one is just rebuild kqueen api-image locally:
+
+  ::
 
    docker build -t kqueen/api:your_tag kqueen/
```

Comment:
It was already wrong before but right command may be `docker build -t kqueen/api:your_tag .` in case we are in the root of this repository.